challenge_spr/src/challenge_spr_states/detect.py

```diff
@@ -21,11 +21,20 @@
 
 
 class DetectCrowd(smach.State):
+    """
+    Detect a crowd and describe it based on individual detections
+    """
     def __init__(self, robot):
         smach.State.__init__(self, outcomes=['succeeded', 'failed'], output_keys=['crowd_data'])
         self.robot = robot
 
-    def execute(self, userdata):
+    def execute(self, userdata=None):
+        """
```

Comment:
Yeah,, that's nice and complete and all but inherited from State already. No need to put this at every subclass. 
Maybe if ```userdata``` required some dict fields to be set etc, that would warrant a docstring but.